torch/csrc/lazy/core/config.cpp

```diff
@@ -50,3 +50,21 @@ C10_DEFINE_string(
     torch_lazy_metrics_percentiles,
     "0.01:0.05:0.1:0.2:0.5:0.8:0.9:0.95:0.99",
     "Metrics percentiles to be collected, using : as the delimiter");
+
+namespace torch {
+namespace lazy {
+
+std::string& getLTCForceFallback() {
```

Comment:
i'm thinking of changing the name to fit in with the `torch_lazy` pattern, but i haven't really thought of what the right name scheme is.  i think we should land this first and then we can consider names later.  my suggestion might be `torch::lazy::get_force_fallback` but i wonder what other people think.  Should 'config' be a namespace, or any other proposal?

Reply:
Both getLTCForceFallback/getForceFallback and get_force_fallback sounds good as long as we are consistent about the flavor.